### ISSUE TITLE
Wrong parameter name for aggregations?

### ISSUE THREAD

**Issue by luctus** [afomera/fathom_api#2]
Hi Andrea, 

First of all, I wanted to say thank you for this gem! I'm using it now and it's super clean.

I believe there is a problem with the param `entity_type` as it looks like it's called just `entity` at Fathom's API. 

If I execute this line:
```rb
client.aggregations.list(entity_id: r.fathom_site_id, entity_type: 'pageview', aggregates: 'visits,pageviews,avg_duration,bounce_rate')
```
I'll get this error:
`Fathom::Error (Your request was malformed. {"errors"=>{"entity"=>["The entity field is required."]}})`

I noticed your gem allows an extra param, so I used it like this:

```rb
client.aggregations.list(entity_id: r.fathom_site_id, entity_type: 'pageview', aggregates: 'visits,pageviews,avg_duration,bounce_rate', entity: 'pageview')
```
And it returned successfuly the data.

**Comment by afomera** [comment]
Ohh this defiantly looks like an issue on our end, or they changed the API endpoint on theirs haha. 

I noticed the aggregations endpoint was a bit weird when I went to use it while building the gem, but figured I was doing something wrong. 

If you feel up to making a PR I'd love a fix for this, if not I'll grab it if I get some time. https://github.com/afomera/fathom_api/blob/main/lib/fathom/resources/aggregations.rb#L3-L7 this is the resource responsible for the aggregation API endpoint. 

Thanks for the detailed report and for using the gem!
